### DISCUSSION TITLE
[i18n] Write a Blog Post that explains how to handle multiple languages with MeiliSearch

### DISCUSSION THREAD

**Discussion by dtruffaut** [meilisearch/product#598] [Feedback & Feature Proposal]
Just like this:

https://www.elastic.co/blog/multilingual-search-using-language-identification-in-elasticsearch

**Comment by CaroFG** [comment]
Hey @dtruffaut,

That's a great idea! We have taken note of it, and we will definitely write about it 🚀

Thank you so much for sharing it! ❤️

**Comment by gmourier** [comment]
Thank you, @dtruffaut and @CaroFG 

I'm locking that discussion then!
